ompi/mca/pml/ob1/pml_ob1_sendreq.h

```diff
@@ -214,6 +215,16 @@ do {
 
 static inline void mca_pml_ob1_send_request_fini (mca_pml_ob1_send_request_t *sendreq)
 {
+
```

Comment:
Do we really want to retest the buffer during completion (it has been already memchecked upon start)?

Reply:
This is more of a catch all for the request functions. I moved it from `mca_pml_ob1_send` to here - maybe we can remove it from the others and just have it here for send requests?

Reply:
I might have missed some subtlety, but I don't see the point of memchecking at the end of the operation